src/oas2/transformers/__tests__/params.test.ts

```diff
@@ -124,6 +124,75 @@ describe('params.translator', () => {
         ),
       ).toMatchSnapshot();
     });
+
+    describe('schema examples', () => {
+      it('given schema examples should translate to body parameter with examples', () => {
+        const body = translateToBodyParameter(
+          {
+            in: 'body',
+            name: 'name',
+            schema: {
+              // @ts-ignore: "x-examples" as schema extension
+              'x-examples': {
+                'example-1': {
+                  hello: 'world',
+                },
+                'example-2': {
+                  foo: 'bar',
+                },
+              },
+            },
+          },
+          consumes,
+        );
+
+        expect(body).toEqual(
+          expect.objectContaining({
+            contents: expect.arrayContaining([
+              expect.objectContaining({
+                examples: [
+                  { key: 'example-1', value: { hello: 'world' } },
+                  { key: 'example-2', value: { foo: 'bar' } },
+                ],
+              }),
+            ]),
+          }),
+        );
+      });
+
+      it('root x-examples should take precedence over schema examples', () => {
```

Comment:
What do you mean by precedence? As in — is the first example listed or Is the second one completely ignored?

Reply:
Meaning schema example ignored if there's parameter example. Though it not said explicitly in oas2 spec as there's no `examples` keyword for body parameter, I followed same approach as oas3 has which is ignoring schema example if there's one in media object: 
> if referencing a schema which contains an example, the example value SHALL override the example provided by the schema.

We may join them if that makes more sense.

Reply:
How about this one? Should examples from root object (body param or response) override schema object examples (like in oas3) or rather be combined altogether?